apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -32,42 +32,51 @@
 %% Type declarations
 %%================================================================================
 
+-ifndef(EMQX_RELEASE_EDITION).
+-define(EMQX_RELEASE_EDITION, ce).
```

Comment:
Just curious: why was this needed?  It should have been set by rebar/mix during compilation.  :thinking:

Reply:
Once again, it's because of the standalone mode.

Reply:
Aha, when testing just the `emqx` app, I see.